models/intel/ssd300-int8-onnx-0001/description/ssd300-int8-onnx-0001.md

```diff
@@ -0,0 +1,50 @@
+# ssd300-int8-onnx-0001
+
+## Use Case and High-Level Description
+
+This is an SSD model with a VGG16 backbone that is designed to perform object detection. The model has been pretrained on the VOC image database (VOC07+12 dataset) and then symmetrically quantized to INT8 fixed-point precision using Neural Network Compression Framework (NNCF).
```

Comment:
If the original public model exists in OMZ, please provide link to it

Reply:
@eizamaliev link to original model not required there

Reply:
@eizamaliev it is not required there